python-checks/src/test/resources/checks/randomSeedSKlearn.py

```diff
@@ -0,0 +1,28 @@
+from sklearn.model_selection import train_test_split
+from sklearn.svm import SVC
+from sklearn.datasets import load_iris, make_blobs
+
+def failure():
+    X, y = load_iris(return_X_y=True)
+    X_train, X_test, y_train, y_test = train_test_split(X, y) # Noncompliant {{Provide a seed for the random_state parameter.}}
+    #                                  ^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Only having the issue on the function name might be clearer, some scikit-learn functions have a lot of parameters. It would make a really large red blob